components/PlayerCard/index.js

```diff
@@ -17,7 +18,7 @@ export default function PlayerCard({ player }) {
         <StyledPlayerLabel>WR:</StyledPlayerLabel>
         <span>{player.worldRanking}</span>
       </div>
-      <StyledLink href={`/players/${player.id}`}>Show Details</StyledLink>
+      <StyledLink variant={'btn-primary'} href={`/players/${player.id}`}>Show Details</StyledLink>
```

Comment:
you don't need the brackets here, since it's a string you pass as a prop. And it is odd to have a Link that works with a prop called "btn-primary", how about just "primary" instead:
```suggestion
      <StyledLink variant='primary' href={`/players/${player.id}`}>Show Details</StyledLink>
```